core/multiwatcher/store.go

```diff
@@ -216,7 +217,7 @@ func (a *store) Get(id EntityID) EntityInfo {
 
 // ChangesSince returns any changes that have occurred since
 // the given revno, oldest first.
-func (a *store) ChangesSince(revno int64) ([]Delta, int64) {
+func (a *store) ChangesSince(revno int64) ([]Delta, int64, error) {
```

Comment:
Why would clone fail?

Reply:
ah, it used to when a reflect based approach was tried but not now. i'll fix